build.sh

```diff
@@ -143,11 +149,12 @@ pixman_dependencies=""
 cairo_dependencies="zlib png pixman"
 xml_dependencies="zlib iconv"
 sqlite_dependencies=""
-openslide_dependencies="png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
+openslide_dependencies="libzip png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
 openslidejava_dependencies="openslide"
 
 # Build artifacts
 zlib_artifacts="zlib1.dll"
+libzip_artifacts="libzip.dll"
```

Comment:
Great.  If you could push that change, I'll merge this PR.

As a rule, OpenSlide avoids passing file descriptors or FILE pointers to other libraries, because on Windows there's no guarantee that those libraries are using the same C runtime that we are.